pkg/apply/task/inv_add_task.go

```diff
@@ -29,13 +29,14 @@ var (
 // into the cluster. The InvAddTask should add/merge inventory references
 // before the actual object is applied.
 type InvAddTask struct {
-	TaskName      string
-	InvClient     inventory.Client
-	DynamicClient dynamic.Interface
-	Mapper        meta.RESTMapper
-	InvInfo       inventory.Info
-	Objects       object.UnstructuredSet
-	DryRun        common.DryRunStrategy
+	TaskName         string
+	ClusterInventory inventory.Inventory
```

Comment:
ClusterInventory name could perhaps be more clear. Is this the inventory, untouched, as it was when we started? Or is it the cached one we update along the way?

"Inventory" might be sufficient.

Reply:
Or if we're not changing it after the Get, maybe `InitialInventory`?

Reply:
Alright, I see we use "Inventory" in the TaskQueueBuilder and "ClusterInventory" in the tasks. I don't think the name ClusterInventory helps clarify what gets stored in it or how to use it. It seems like it starts as what was on the cluster, but it could become stale and then we modify it. So we might as well just call it "Inventory" everywhere and be clear in the struct field comments.